docs/api/tuf.api.metadata.supporting.rst

```diff
@@ -0,0 +1,27 @@
+Supporting classes
+---------------------------------
+
+The Metadata API includes multiple classes that are used by the top-level
+ones (Root, Timestamp, Snapshot, Targets):
+
+.. autosummary::
+   :nosignatures:
+
+   tuf.api.metadata.DelegatedRole
+   tuf.api.metadata.Delegations
+   tuf.api.metadata.Key
+   tuf.api.metadata.MetaFile
+   tuf.api.metadata.Role
+   tuf.api.metadata.TargetFile
+
+.. autoclass:: tuf.api.metadata.DelegatedRole
+
+.. autoclass:: tuf.api.metadata.Delegations
+
+.. autoclass:: tuf.api.metadata.Key
+
+.. autoclass:: tuf.api.metadata.MetaFile
+
+.. autoclass:: tuf.api.metadata.Role
+
+.. autoclass:: tuf.api.metadata.TargetFile
```

Comment:
In my opinion, the order of definition could be better.
Starting with the Root helper classes (as the first of the four metadata classes defined in metadata.py) - `Key` and `Role`, then going with Timestamp and Snapshot helper class - `MetaFile` and finish with Targets helper classes `Delegations` and `DelegatedRole`.